### PR TITLE
Example level improvements

### PR DESCRIPTION
**Pull request by Th3Fanbus**
Please see individual commit messages.

### CODE REVIEW

**Comment by budak7273** [comment]
Requires #310 to be merged first otherwise the buildings are inside the block

File: Source/FactoryGame/Private/FGWorldScannableData.cpp [hunk 1]
```diff
@@ -1,18 +1,163 @@
...
+#include "Buildables/FGBuildableFrackingActivator.h"
+#include "FGScannableSubsystem.h"
+#include "FGItemPickup.h"
+#include "FGDropPod.h"
+#include "WorldPartition/WorldPartition.h"
+#if WITH_EDITOR
+#include "WorldPartition/WorldPartitionActorDesc.h"
+#endif
+
+FWorldScannableData::FWorldScannableData()
```
Comment (Th3Fanbus) [inline Source/FactoryGame/Private/FGWorldScannableData.cpp:14]: I had to add this in order to make it build, but the latest Satisfactory update made this redundant. So this needs to be dropped.

File: Source/FactoryGame/Private/FGWorldScannableData.cpp [hunk 2]
```diff
@@ -1,18 +1,163 @@
...
-#include "UObject/ObjectSaveContext.h"
+#include "EngineUtils.h"
+#include "Buildables/FGBuildableFrackingActivator.h"
+#include "FGScannableSubsystem.h"
+#include "FGItemPickup.h"
+#include "FGDropPod.h"
+#include "WorldPartition/WorldPartition.h"
+#if WITH_EDITOR
+#include "WorldPartition/WorldPartitionActorDesc.h"
+#endif
```
Comment (budak7273) [inline Source/FactoryGame/Private/FGWorldScannableData.cpp:12]:
```suggestion

```

Comment (Th3Fanbus) [reply Source/FactoryGame/Private/FGWorldScannableData.cpp:12]: Discussed elsewhere, my comment was about the constructor. I don't know how to GitHub.

**Comment by budak7273** [comment]
Mircea brought the custom implementations over into the header implementer https://discord.com/channels/555424930502541343/562722670974599227/1304892216392552581

**Review by budak7273 — APPROVED** [review]

**Pull request merged**